lib/train/transports/azure.rb

```diff
@@ -35,11 +35,15 @@ def initialize(options)
         if @options[:client_secret].nil? && @options[:client_id].nil?
           parse_credentials_file
         end
+
+        # additional platform details
+        @platform_details = { release: '0.15' }
```

Comment:
We might need to clarify that this is the azure_mgmt_resource gem version, NOT the Azure API version.  The Azure API allows you to use different API versions for different services, and the gem supports that.  So you could talk to the VM service at version 2017-07-03, and the AD service at version 2018-02-05 (those are arbitrary, but they are named after dates)